py-package/{{cookiecutter.project_slug}}/.github/workflows/publish.yml

```diff
@@ -9,19 +9,26 @@ on:
 
 jobs:
   deploy:
-
     runs-on: ubuntu-latest
-
+    environment:
+      name: pypi
+      url: https://pypi.org/p/${{ github.repository }}
+    permissions:
+      id-token: write # IMPORTANT: this permission is mandatory for trusted publishing
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v2
+    - uses: actions/checkout@v4
+    - name: Install uv and Python
+      uses: astral-sh/setup-uv@v5
       with:
-        python-version: '3.x'
-    - name: Install dependencies
+        version: "latest"
```

Comment:
Is this the recommended way of setting up the cache with uv

Reply:
I guess so, I copied it somewhere from you or from uv, I guess :D